master/internal/job/job.go

```diff
@@ -124,134 +137,117 @@ func (j *Jobs) getJobs(
 	return jobsInRM, nil
 }
 
-func (j *Jobs) setJobPriority(ctx *actor.Context, jobID model.JobID, priority int) error {
+func (j *Jobs) setJobPriority(ref *actor.Ref, priority int) error {
 	if priority < 1 || priority > 99 {
 		return errors.New("priority must be between 1 and 99")
 	}
-	jobActor := j.actorByID[jobID]
-	resp := ctx.Ask(jobActor, sproto.SetGroupPriority{
+	resp := j.system.Ask(ref, sproto.SetGroupPriority{
```

Comment:
the only thing weird about this is that if the actor is closed, it just returns nil, but that is probably fine behavior for this anyway